apis/nucleus/src/object/navigation/navigation.js

```diff
@@ -0,0 +1,54 @@
+export default function createNavigationApi(halo, store) {
+  const { galaxy } = halo.public;
+  if (galaxy.anything?.sense?.navigation) {
+    return galaxy.anything?.sense?.navigation;
+  }
+  const State = {};
+
+  /**
+   * @class Navigation
+   */
+  const navigationAPI = /** @lends Navigation# */ {
+    /**
+     * Navigate to the supplied sheet
+     * @param {string} sheetId Id of the sheet to navigate to
+     */
+    goToSheet: async (sheetId) => {
+      const { modelStore, rpcRequestModelStore } = store;
+      const key = `${sheetId}`;
+      let rpc = rpcRequestModelStore.get(key);
+      if (!rpc) {
+        rpc = halo.app.getObject(sheetId);
+        rpcRequestModelStore.set(key, rpc);
+      }
+      const model = await rpc;
+      modelStore.set(key, model);
+      State.sheetRef?.current?.setModel?.(model);
```

Comment:
maybe add code for if sheetId is not a valid sheet id (not an object, or an object that is not a sheet)
(here or maybe in Sheet.jsx)

Reply:
Fixed!